lms/djangoapps/mobile_api/course_info/urls.py

```diff
@@ -4,16 +4,11 @@
 from django.conf.urls import patterns, url
 from django.conf import settings
 
-from .views import CourseAboutDetail, CourseUpdatesList, CourseHandoutsList
+from .views import CourseUpdatesList, CourseHandoutsList
 
 urlpatterns = patterns(
     'mobile_api.course_info.views',
     url(
```

Comment:
@mhoeber Please note that this PR removes support for the CourseAbout MobileAPI endpoint: http://edx.readthedocs.org/projects/edx-platform-api/en/latest/mobile/course_info.html#get-the-course-about-page

@lamagnifica FYI.